sets.tex

```diff
@@ -0,0 +1,86 @@
+%% Sets
+
+SymPy supports representation of a wide variety of sets, this is achieved by
+first defining abstract representation for a smaller number of atomic set
+classes and then combining and transforming them using various set operations.
+
+Each of the set classes inherit from the base set class and defines rules to
```

Comment:
@shivamvats I feel it will be inherit only. Because then it'll be two plurals together, which won't be grammatically correct.